CONTRIBUTING.rst is outdated

# Bug report

## What's wrong

In CONTRIBUTING.rst there is the section *Performance* where it is suggested to run `make benchmarks` but this command seems to be dropped:

    $ make benchmarks
    make: *** No rule to make target 'benchmarks'.  Stop.

In the next section *Type checking* the suggested command is wrong:

    $ make type-checking
    make: *** No rule to make target 'type-checking'.  Stop.

The working command is:

    $ make type-check
    mypy mimesis/ tests/

--- a/CONTRIBUTING.rst
+++ b/CONTRIBUTING.rst
@@ -188,19 +188,6 @@ To run tests, simply:
 Check out logs of Travis CI or AppVeyor if tests were failed on creating
 PR, there you can find useful information.
 
-Performance
-~~~~~~~~~~~
-
-It’s good idea to run benchmark test, when you add your feature:
-
-.. code:: text
-
-    ⟩ make benchmarks
-
-Optimize the things which really must be optimized. There no need in
-using ``C`` or other overheads to win 0.0000001 seconds of runtime.
-
-
 Type checking
 ~~~~~~~~~~~~~
 

--- a/CONTRIBUTING.rst
+++ b/CONTRIBUTING.rst
@@ -209,7 +209,7 @@ sure that everything is okay. You can do it using make:
 
 ::
 
-    ⟩ make type-checking
+    ⟩ make type-check
 
 Code Review
 ~~~~~~~~~~~

--- a/CONTRIBUTORS.rst
+++ b/CONTRIBUTORS.rst
@@ -77,6 +77,7 @@ Contributors
 -  Jack Henry `(jackhenry)`_
 -  Nikita Serba `(Nekit10)`_
 -  Julien Surloppe `(jsurloppe)`_
+-  Emilio Cecchini `(ceccoemi)`_
 
 .. _(lk-geimfari): https://github.com/lk-geimfari
 .. _(sobolevn): https://github.com/sobolevn
@@ -130,3 +131,4 @@ Contributors
 .. _(jackhenry): https://github.com/jackhenry
 .. _(Nekit10): https://github.com/Nekit10
 .. _(jsurloppe): https://github.com/jsurloppe
+.. _(ceccoemi): https://github.com/ceccoemi

